tool/jt.rb

```diff
@@ -1043,11 +1054,10 @@ def test_cexts(*args)
         case gem_name
         when 'backtraces'
           run_ruby "-I#{dir}/lib", "#{dir}/bin/#{gem_name}", err: output_file, continue_on_failure: true
-          actual = File.read(output_file).gsub(TRUFFLERUBY_DIR, '').gsub(/\/cext\.rb:(\d+)/, '/cext.rb:n')
+          actual = File.read(output_file).gsub(TRUFFLERUBY_DIR, '').gsub(/\/cext(_ruby)?\.rb:(\d+)/, '/cext\1.rb:n')
```

Comment:
It would be much cleaner to use a capture group around the entire filename, and make the inner group non-capturing:
```ruby
gsub(/(\/cext(?:_ruby)?\.rb):(\d+)/, '/\1:n')
```

Reply:
I don't agree the original version has less parenthesis and seems easier to read to me. This is unimportant change I am keeping the original.

Reply:
OK.